pkg/apis/authenticator.aws/register.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2017 by the contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package authenticatoraws
+
+const (
+	// GroupName represents the base of this package
+	GroupName = "authenticator.aws"
```

Comment:
Most of the kubernetes APIs use something like `x.y.k8s.io` as the GroupName.  In other areas, I've used amazonaws.com as the DNS label in situations where the community uses DNS labels, since its actually owned by AWS and the point is to use something you own as a namespace to reduce collisions.  In this case, I think the namespace includes any other CRDs that they create?  TLDR: My suggestion is to use `authenticator.amazonaws.com` as the GroupName, mostly because my assumption is that its more consistent.

Reply:
Cool, JFYI we do own the TLD `aws` we use it for services like Chime… https://app.chime.aws for example. Was hoping we could get something short since it ends up being the full resource if you tab complete `kubectl get iamidentitymapping` turns into `kubectl get iamidentitymapping.authenticator.aws` with `authenticator.amazonaws.com` it will just make it really long as `iamidentitymapping.authenticator.amazonaws.com`

Reply:
:+1: